classes/game.rb

```diff
@@ -0,0 +1,92 @@
+class Game
+  attr_reader :game_name, :player1, :player2, :players, :turn_of, :table
+
+  def initialize (game_name, player1, player2)
+    @game_name = game_name
+    @player1 = player1
+    @player2 = player2
+    @players = [player1, player2]
+
+    @game_set = ['_', '_', '_', '_', '_', '_', '_', '_', '_']
+    @victory_combinations = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [6, 4, 2]
+    ]
+
+    @turn_of = random_initial_turn_of
+    @table = set_table
+  end
+
+  def start_game
+    i=0
+
+    while i<9 do
+      i+=1
+      choice_of_player
+    end
+  end
+
+  
+  private
+
+  def set_table
+    table = ''
+
+    @game_set.each_with_index do |value, index|
+      table += value if [0, 3, 6].include?(index)
+      table += '|' + value + '|' if [1, 4, 7].include?(index)
+      table += value + "\n" if [2, 5, 8].include?(index)
+    end
+
+    puts(table)
+    table
+  end
+  
+  def change_turn
+    return @turn_of = 0 if @turn_of == 1
+    return @turn_of = 1
+  end
+
+  def random_initial_turn_of
```

Comment:
These methods should be named as verbs because they do something
```suggestion
  def choose_initial_turn
```